examples/causal_inference/interventional_distribution.ipynb

```diff
@@ -0,0 +1,1561 @@
+{
```

Comment:
<p>Similarly a bit quick into the mechanics of the do-operator as a graph mutilation device. Better to set up the idea more slowly. Lead the audience to the idea from : it'd be great to examine counterfactuals to counterfactuals can be seen as interventions on a network graph of causal flow. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Done. I've added quite a bit of extra text to help with this</p>